### DISCUSSION TITLE
一点感想

### DISCUSSION THREAD

**Discussion by Albert-Ma** [mli/paper-reading#6] [General]
早上看到沐神的这个repo，看了下边的文章都很经典，抱着试一试的态度哈哈，从B站上花了俩小时竟然看完了（2x倍速 dog :) ），有一点感想，写在这里

我觉得跟沐神读这些经典论文，除了让我们了解了这些经典论文的思想，论文精读时的一些细节，当然还时不时有沐神穿插着讲点深度学习上的一些自己的理解，趣事，都让我受益匪浅。今天上午从沐神视频讲解里，我同样也看到了这些经典论文的motivation，他们为什么做这件事，他们是怎么想出来这么做的，比如ResNet，为什么网络越深会变差，如果浅层网络后边跟着的网络学的跟前边一样，也不至于变差，怎么纠正他们学偏呢？把前边这个思想用过来，让深层的网络最差也跟浅层的网络学的东西一样，他不就不会变差了吗？自己的一些理解。

所以我觉得再看论文的时候 跟着沐神听的时候 认真思考这些经典论文的想法来源，一些思路是很重要的，也是我今天上午收获最大的地方

感谢沐神，很有意义的这个系列

**Comment by jizhang02** [comment]
同意楼上的朋友的说法，这样的系列真的是受益匪浅。或许可以集思广益，开通一个小论坛，把大家集中想问的问题，做一个讲解。作为回馈，如果我们觉得有用，会自发的请您喝咖啡。 😄 
Agree with my friend upstairs that a series like this would really be beneficial. Maybe we can brainstorm and open a small forum to explain the questions we all want to ask in a focused manner. In return, if we find it useful, we will spontaneously buy you a coffee. 😄

**Reply by mli** [reply]
可以考虑哈。先做几期内容，如果大家感兴趣的话可以搞个qa环节
